packages/app/src/device/listeners/register-external-usage-device.listner.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { createClient } from "App/__deprecated__/api/mudita-center-server/create-client"
+import { ipcMain } from "electron-better-ipc"
+import { ExternalDataApiEvent } from "App/device/constants/external-data-api-event.constant"
+
+const registerExternalUsageDevice = (): void => {
+  const client = createClient()
+
+  ipcMain.answerRenderer(
+    ExternalDataApiEvent.GetExternalUsageDevice,
+    (serialNumber: string) => {
+      return client.getExternalUsageDevice(serialNumber)
+    }
+  )
+}
+
+export default registerExternalUsageDevice
```

Comment:
🟢 (a good candidate for small PR)
This is a deprecated approach. Please try to rewrite the `listener` to controller & service as a standalone module. I think that this request could merge with the first comment as a good candidate for a small refactor as a standalone PR.